pegasus/sites.v3/hourofcode.com/public/map.haml

```diff
@@ -13,5 +14,11 @@ layout: wide_index
 
   = view :header
 %h2.center-text= view :hoc_events_counter
-= view :hoc_events_map
-.footnote!=hoc_s(:map_warning_markdown, markdown: :inline, locals:{events_url: resolve_url('/events')})
+
+#maparea.full-width{style: "margin-top: 2rem"}
+  -# Show a picture of the map unless it is currently the week of HOC, or the user is located in Morocco (border disputes misrepresented in map)
+  -if hoc_mode == "actual-hoc" || @country == 'ma'
```

Comment:
just confirming my understanding that we are ok with showing non-Moroccan countries the Moroccan world view during HOC, because we are highly confident that the Kosovo issue is the only thing controversial about that map? I'm not sure what other visible differences there are, but I am commenting because we'll both be getting a lot of traffic when the static map is shown to all viewers during the week of HOC, and this will also be a high stakes time to make changes to that page.

Reply:
This is a very fair question, and one I'd like other folks to weigh in on if possible. The reason I am hopeful about this approach is that the map is so zoomed out, it is unlikely that disputed borders will be visible (which is the map layer the Moroccan filter affects). However, I am not sure how else to know for sure.

Reply:
My thoughts on this... We'd make a change to update the static image before HOC with all the events. I wonder if we can have two images, one for defacto and one for Morocco that can be selected based on the geo location...? This could be very manual if the number of countries increase, but seems safest to prevent any last minute changes during HoC.

Reply:
Yes, that seems safest. I can update our ticket to update the static map with this plan to differentiate for MA!

Reply:
Thanks all for considering, this sounds great to me!